docs/RELEASE-PROCESS.md

```diff
@@ -0,0 +1,87 @@
+# Release process
+
+## Determine the new version
+Extension versions may not have suffixes like "-rc1."
```

Comment:
```suggestion
Extension versions may not have suffixes like "-rc1".
```

Reply:
seeing as this is in the oasisprotocol org and many of its people are in europe, fine

Reply:
Oh interesting I didn't know this punctuation inside/outside quotes was a thing, it just looked wrong to me as it looked like the dot was part of the suffix :-)